### PR TITLE
Remove @todo, fix a small bug in connection processor

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This PR removes a `@todo` from the connection processor and fixes a
small bug in the code of the same file.

Related to #467.

### CODE REVIEW

File: src/connection/connbuf.c [hunk 1]
```diff
@@ -139,3 +139,16 @@ ws_connbuf_discard(
     return 0;
 }
 
+char*
+ws_connbuf_buffer(
+    struct ws_connbuf* self //!< The object
```
Comment (TheNeikos) [inline src/connection/connbuf.c]: Doc string in implementation?

**Comment by neithernut** [comment]
ACK for squashing.

**Comment by matthiasbeyer** [comment]
Done.

**Comment by neithernut** [comment]
ACK.

**Pull request merged**